packages/did/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@atproto/did",
+  "version": "0.0.1",
+  "license": "MIT",
+  "description": "DID resolution and verification library",
+  "keywords": [
+    "atproto",
+    "did",
+    "validation",
+    "types"
+  ],
+  "homepage": "https://atproto.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bluesky-social/atproto",
+    "directory": "packages/did"
+  },
+  "type": "commonjs",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    }
+  },
+  "dependencies": {
+    "tslib": "^2.6.2",
```

Comment:
due to our configuration of ts, it might get used depending on EcmaScript features that we reply upon. You are right though that no file from `dist` uses them. I'll drop this dep.

Reply:
I used to have `importHelpers` on in the other repo when I started working on oauth. I believe that using it would reduce our bundle size quite a bit but I'll leave it off for now, removing the tslib dep.